pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -173,57 +172,106 @@ func NewFakeFilterPlugin(failedNodeReturnCodeMap map[string]framework.Code) fram
 	}
 }
 
-func numericMapPriority(pod *v1.Pod, meta interface{}, nodeInfo *schedulernodeinfo.NodeInfo) (framework.NodeScore, error) {
+type numericMapPlugin struct{}
+
+func NewNumericMapPlugin() framework.PluginFactory {
```

Comment:
we don't need to export this or any of the functions below (apart from the plugin interface, like Score and Name)

Reply:
Updated